libs/docs/platform/table/e2e/table.e2e-spec.ts

```diff
@@ -172,7 +172,8 @@ describe('Table component test suite', () => {
             await findElementInTable(tableDefaultExample, tableCellArr);
         });
 
-        it('should check correct operation x button', async () => {
+        // Simple example was updated to use custom heading.
```

Comment:
Custom heading shouldn't break this test, we should just change it to get the correct button to pass the test